rules/coding-style/src/Rector/Variable/UnderscoreToPascalCasePropertyNameRector.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\CodingStyle\Rector\Variable;
```

Comment:
Is it a good idea to move this rule to new `Rector\CodingStyle\Rector\Property` namespace?

Reply:
Yes